src/test/js/multibranch/testResults.js

```diff
@@ -40,6 +40,7 @@ module.exports = {
         blueRunDetailsPage.clickTab('tests');
 
         // Expand the test.
+        browser.useXpath().waitForElementVisible('//div[@class="result-item-head"]');
```

Comment:
Yeah ... please put these paths into Page Object selectors and hide all the XPath shit. I'm obviously not against the use of XPath (sometimes no option), but I hate seeing it in the test code.